apps/deploy-web/src/utils/apiUtils.ts

```diff
@@ -127,7 +127,7 @@ export class ApiUrlService {
 
 // TODO: implement proper pagination on clients
 //   Issue: https://github.com/akash-network/console/milestone/7
-export async function loadWithPagination<T>(baseUrl: string, dataKey: string, limit: number) {
+export async function loadWithPagination<T>(baseUrl: string, dataKey: string, limit: number, httpClient = axios) {
```

Comment:
dirty hack to pass httpClient from context, so then it can be replaced in unit tests. "dirty" hack because it's a temporary solution until we refactor to services which is provider via single context